surfer/viz.py

```diff
@@ -932,6 +932,10 @@ def add_overlay(self, source, min=2, max="robust_max", sign="abs",
             If None, it is assumed to belong to the hemipshere being
             shown. If two hemispheres are being shown, an error will
             be thrown.
+        **kwargs: additional keyword arguments
+            These are passed to the underlying
+                :func:`mayavi.mlab.pipeline.surface` call.
```

Comment:
This extra indentation makes for weird spacing, see CircleCI doc artifact:
```suggestion
            :func:`mayavi.mlab.pipeline.surface` call.
```

https://292-1551431-gh.circle-artifacts.com/0/html/generated/surfer.Brain.html#surfer.Brain.add_foci

And error:

Also many links do not work:
```
/home/circleci/project/surfer/viz.py:docstring of surfer.Brain.add_annotation:37: WARNING: py:func reference target not found: mayavi.mlab.pipeline.surface
/home/circleci/project/surfer/viz.py:docstring of surfer.Brain.add_contour_overlay:42: WARNING: py:func reference target not found: mayavi.mlab.pipeline.contour_surface
/home/circleci/project/surfer/viz.py:docstring of surfer.Brain.add_data:105: WARNING: py:func reference target not found: mayavi.mlab.pipeline.surface
/home/circleci/project/surfer/viz.py:docstring of surfer.Brain.add_foci:41: WARNING: py:func reference target not found: mayavi.mlab.point3d
/home/circleci/project/surfer/viz.py:docstring of surfer.Brain.add_label:43: WARNING: py:func reference target not found: mayavi.mlab.pipeline.surface
/home/circleci/project/surfer/viz.py:docstring of surfer.Brain.add_morphometry:33: WARNING: py:func reference target not found: mayavi.mlab.pipeline.surface
/home/circleci/project/surfer/viz.py:docstring of surfer.Brain.add_overlay:29: WARNING: py:func reference target not found: mayavi.mlab.pipeline.surface
```
These should just be put in double-backticks to render as code. Or do the `intersphinx` dump and find a link that exists in the `objects.inv`.

Reply:
At least one is just a typo, `point3d` should be `points3d` so it directs here

https://docs.enthought.com/mayavi/mayavi/auto/mlab_helper_functions.html#points3d

Fixing these errors is required to get CircleCI happy

Reply:
Sorry for the late reply. I was a little overloaded these days. I just pushed a new commit. And, gladly, the errors seem to be solved.